src/buttons/src/Button.js

```diff
@@ -27,12 +27,34 @@ const ButtonIcon = memo(function ButtonIcon({ icon, size, spacing, edge }) {
   )
 })
 
-const styles = {
-  position: 'relative',
-  fontWeight: 500,
+export const sizes = {
+  small: {
+    height: 24,
+    textSize: 300
+  },
+  medium: {
+    height: 32,
+    textSize: 300
+  },
+  large: {
+    height: 40,
+    textSize: 400
+  }
+}
```

Comment:
I don't love this. It should probably move into the theme hook.

Reply:
+1 - what if we wanted to let users configure these values? I like the size API, but as a user, I'd likely want to control  what `small` `medium` and `large` map to as sizes for height / text size. 

Should these be semantic tokens? I understand the APIs aren't super intuitive because we use `<Text />` underneath (which expects `size`), but I think that's something we can solve (at least for now) via documentation + types.